.devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/dotnet/.devcontainer/base.Dockerfile
+
+# [Choice] .NET version: 5.0, 3.1, 2.1
+ARG VARIANT="5.0"
```

Comment:
Is this good enough?

Reply:
The version of dotnet installed on the container is largely irrelevant since we want to use the locally installed SDK for almost everything in the repo so defaulting to the current RTM should be sufficient.

Reply:
Got it